docs/apache-airflow/howto/dynamic-dag-generation.rst

```diff
@@ -29,7 +29,7 @@ If you want to use variables to configure your code, you should always use
 `environment variables <https://wiki.archlinux.org/title/environment_variables>`_ in your
 top-level code rather than :doc:`Airflow Variables </concepts/variables>`. Using Airflow Variables
 at top-level code creates a connection to metadata DB of Airflow to fetch the value, which can slow
-down parsing and place extra load on the DB. See the `Airflow Variables <_best_practices/airflow_variables>`_
+down parsing and place extra load on the DB. See the :doc:`Airflow Variables <_best_practices/airflow_variables>`
```

Comment:
```suggestion
down parsing and place extra load on the DB. See the :doc:`Airflow Variables <best_practices/airflow_variables>`
```

Leading underscore is a special restructuredText syntax (reference) and not a part of the actual object name.